src/measure_theory/integral/circle_integral_transform.lean

```diff
@@ -0,0 +1,173 @@
+/-
+Copyright (c) 2022 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import analysis.complex.cauchy_integral
+import analysis.analytic.basic
+import analysis.calculus.parametric_interval_integral
+import data.complex.basic
+import measure_theory.integral.circle_integral
+/-!
+# Circle integral transform
+
+In this file we define the circle integral transform of a function `f` with complex domain. This is
+defined as $(2πi)^{-1}\frac{f(x)}{x-w}$ where `x` moves along a circle. We then prove some basic
+facts about these functions.
+
+These results are useful for proving that the uniform limit of a sequence of holomorphic functions
+is holomorphic.
+
+-/
+
+open set measure_theory metric filter function
+open_locale interval real
+
+noncomputable theory
+
+variables {E : Type} [normed_group E] [normed_space ℂ E] (R : ℝ) (z w : ℂ)
+
+namespace complex
+
+/-- Given a function `f : ℂ → E`, this gives the function $(2πi)^{-1}\frac{f(x)}{x-w}$ where `x`
+runs over a circle of radius `R` around `z`. If `f` is differentiable and `w` is in the interior of
+the ball, then the integral from `0` to `2 * π` of this gives the value `f(w)`. -/
+def circle_integral_transform (f : ℂ → E) : ℝ → E :=
+  λ θ, (2 * ↑π * I)⁻¹ • deriv (circle_map z R) θ • ((circle_map z R θ) - w)⁻¹ • f (circle_map z R θ)
```

Comment:
```suggestion
def circle_integral_transform (f : ℂ → E) (θ : ℝ) : E :=
(2 * ↑π * I)⁻¹ • deriv (circle_map z R) θ • ((circle_map z R θ) - w)⁻¹ • f (circle_map z R θ)
```
Most of the time, it is more readable to define a function by putting all its arguments before the colon.